diff --git a/target/linux/lantiq/xrx200/base-files/etc/board.d/02_network b/target/linux/lantiq/xrx200/base-files/etc/board.d/02_network
index 3122d40c92..090fd09f32 100644
--- a/target/linux/lantiq/xrx200/base-files/etc/board.d/02_network
+++ b/target/linux/lantiq/xrx200/base-files/etc/board.d/02_network
@@ -15,7 +15,8 @@ lantiq_setup_interfaces()
 	arcadyan,vg3503j)
 		ucidef_set_interface_lan "lan1 lan2"
 		;;
-	arcadyan,arv7519rw22)
+	arcadyan,arv7519rw22|\
+	bintec,rs353)
 		ucidef_set_interface_lan "lan1 lan2 lan3 lan4 lan5"
 		;;
 	arcadyan,vgv7510kw22-brn|\
